nengo/builder/probe.py

```diff
@@ -33,10 +33,8 @@ def synapse_probe(model, key, probe):
         raise ValueError("Attribute '%s' is not probable on %s."
                          % (key, probe.obj))
 
-    if isinstance(probe.slice, slice):
+    if probe.slice is not None:
         sig = sig[probe.slice]
-    else:
```

Comment:
[`git blame`](https://github.com/nengo/nengo/blame/99312f30b5720e297e855a5d6b46c1808b5de6f8/nengo/builder.py#L1243) says this was @hunse's doing, so he can comment on it... seems fine to me!